lib/ansible/modules/network/enos/enos_config.py

```diff
@@ -38,7 +38,7 @@
     a deterministic way.
 extends_documentation_fragment: enos
 notes:
-  - Tested against ENOS 8.4.1.2
+  - Tested against ENOS 8.4.1.0
```

Comment:
Just making a note here as this is the only one left that has a fourth digit

Reply:
Changing now